model/timespan.go

```diff
@@ -12,6 +12,7 @@ type TimeSpan struct {
 	OffsetUTC     int
```

Comment:
Yeah, feel free to do the UI part as well. I'd say the UI could look like this, the notes should be defined as Markdown and then are rendered like this inside the ui.
![image](https://user-images.githubusercontent.com/14895212/100544571-a579e880-324e-11eb-827a-d567ae8f9a2f.png)

Reply:
The note can then be edited by clicking it.

Reply:
Do you want to include this in the current pr or another?

Reply:
I'll see if I can come up with something. Currently this PR breaks the client due to the required notes input field